packages/web-config-server/src/export/excelFormatters/formatMatrixDataForExcel.js

```diff
@@ -124,12 +127,22 @@ const convertAoaToAoo = data => {
 
 const isEmpty = data => data === undefined || data === null;
 
-const addValueOrEmpty = value => {
+const addValueOrEmpty = (value, valueType) => {
   if (isEmpty(value)) return '';
 
   if (typeof value === 'object') {
+    if (valueType === VALUE_TYPES.NUMBER_AND_PERCENTAGE) {
```

Comment:
Do you think we should make this similar to the pattern in `formatter.js` (check line 102)?

```
const VALUE_TYPE_TO_FORMATTER = {
  [VALUE_TYPES.NUMBER_AND_PERCENTAGE]: numberAndPercentage,
};
```